Things.md

```diff
@@ -178,4 +180,16 @@ Classes are generated using [ObjectMakr](https://github.com/FullScreenShenanigan
 
 When a FullScreenShenanigans game starts up, ObjectMakr receives the inheritance structure and properties list from an `objects.js` file. It then generates prototypes for all of the internal classes (Box, Tree, etc.). Most of the basic properties, like `height`, `width`, and `opacity` are defined in the parent Thing class. ObjectMakr's `addThing` is a shortcut to initializing a new instance of a Thing class and assigning properties to it in one step. 
 
-More can be read about ObjectMakr on its [ReadMe](https://github.com/FullScreenShenanigans/ObjectMakr/blob/master/README.md).
\ No newline at end of file
+More can be read about ObjectMakr on its [ReadMe](https://github.com/FullScreenShenanigans/ObjectMakr/blob/master/README.md).
+
+# Sprites
+
+Sprites comprise any visual you see in a FullScreenShenanigans game. They are stored as string representations of an image, containing information about every pixel. The game generates the image from this string in real-time.
+
+Each Thing class has a set of sprites relevant to it, usually different views (facing forward, to the left, etc.). The game selects which sprite to use based on the Thing's full class nomenclature (i.e. if the Player is facing left, its class could be "Player left").
+
+## Sprite Cycles and Movement
+
+A sprite cycle is a timed change in a Thing's class handled by TimeHandlr. This is represented by a cycling through of selected sprites, where each phase lasts a specified time interval. 
+
+Sprite cycles are used to simulate movement. For example, you can have a character walk by adding a cycle between the walking and standing sprites.
```

Comment:
> have a character walk

Small nit: it would be better to mention this as a visual thing, like saying you can show the character walking, or that you can make it look like it's walking.